qa/README.md

```diff
@@ -0,0 +1,17 @@
+# Neural Search QA
+
+## Model Deployment Guidelines
+
+### Important Notes for Model Handling
+
+1. **Non-ML Node Deployment**
+   - To deploy models on non-ML nodes, set `plugins.ml_commons.only_run_on_ml_node` to `false`
+   - **Warning**: Resetting this setting will undeploy all currently deployed models
+
+2. **Avoid Duplicate Deployments**
```

Comment:
Discussion: Instead of adding note doc here, are we able to only allow the sharing model being deployed and used in code? So that new tests added by developers anyway will be limited to use the shared models rather than potential new model resources.